L1Trigger/CSCTriggerPrimitives/python/cscTriggerPrimitiveDigis_cfi.py

```diff
@@ -510,6 +510,17 @@
                            )
 
 ## GEM-CSC ILT in ME1/1
+from Configuration.Eras.Modifier_run2_GEMSliceTest_cff import run2_GEMSliceTest
+run2_GEMSliceTest.toModify( cscTriggerPrimitiveDigis,
+                        GEMPadDigiProducer = cms.InputTag("simMuonGEMPadDigis"),
+                        commonParam = dict(
+                            isSLHC = cms.bool(True),
+                            smartME1aME1b = cms.bool(True),
+                            runME11ILT = cms.bool(True)),
+                        clctSLHC = dict(clctNplanesHitPattern = 3),
+                        me11tmbSLHCGEM = me11tmbSLHCGEM
```

Comment:
I doubt that a fully working integrated trigger will be implemented and tested by the slice test. I will check with the UCLA colleagues.